tools/clang/test/CodeGenSPIRV/rich.debug.local-variable.hlsl

```diff
@@ -25,8 +26,7 @@
 // CHECK: [[tempLexBlock:%\d+]] = OpExtInst %void [[set]] DebugLexicalBlock [[source]] 37 5 [[ifLexBlock]]
 // CHECK:              {{%\d+}} = OpExtInst %void [[set]] DebugLocalVariable [[varNameB]] [[uintType]] [[source]] 38 12 [[tempLexBlock]] FlagIsLocal
 
-float4 main(float4 color : COLOR) : SV_TARGET
-{
+float4 main(float4 color : COLOR) : SV_TARGET {
```

Comment:
I would intentionally put the "{" on the next line (line 30). So we can CHECK the DebugFunction to be on line 29, and DebugLexicalBlock to be on line 30. This could prevent some future bugs.